src/Utils.jl

```diff
@@ -23,6 +23,56 @@ function suffix(T::DataType)
     return "FP64"
 end
 
+function get_GrB_Type(T::DataType)
```

Comment:
This looks like a case for multiple dispatch or a lookup table

```julia
const grb_type = Dict(
   Bool => GrB_BOOL,
   Int8 => GrB_INT8,
...
)

then `grb_type[Bool]` to access it. 

Oe refactoring the client code to call `get_GrB_Type(v::T)` and then dispatching on `T` rather than calling a function on a type directly.

Reply:
Dictionaries are slow, so I would not use the here.

You can do multiple dispatch with `get_GrB_Type(::Type{Bool}) = Grb_BOOL`